BlueShift-iOS-SDK/BlueshiftEventAnalyticsHelper.m

```diff
@@ -41,6 +42,16 @@ + (NSDictionary *)getTrackingParamsForNotification:(NSDictionary *)details {
             //Check for adapter uuid in inapp payload
             adapterId = [details objectForKey:kBSAccountAdapterUUID];
         }
+        //Check execution_key at root level/in data object
+        if (execution_key) {
+            [trackingParams setObject:execution_key forKey: kBSTrackingEK];
+        } else if ([details objectForKey:kBSExecutionContext]){
+            // if not found for in-app, then check into execution_context
+            execution_key = details[kBSExecutionContext][kBSContext][kBSExecutionKey];
```

Comment:
Should we add a nil check for the `context` inside `execution_context`? By chance, if it comes as nil, this line might crash, right?

Reply:
If context is not present, it doesn't crash but returns `nil`. In case  kBSContext or kBSExecutionKey value is nil, it will return nil.